docs/pages/access-controls/sso/github-sso.mdx

```diff
@@ -415,3 +421,31 @@ After logging out of `tsh`, you can log back in without specifying
 
 (!docs/pages/includes/sso/loginerrortroubleshooting.mdx!)
 
+### Teams Not Mapping to Roles
+
+If you are not seeing teams mapping to roles confirm you 
+are using the slug value of the organizations and teams in the connector.
+The GitHub Web application URIs and GitHub API can provide the slug values.
```

Comment:
I think we should include the actual text from https://docs.github.com/en/rest/teams/teams?apiVersion=2022-11-28#get-a-team-by-name

> To create the slug, GitHub replaces special characters in the name string, changes all words to lowercase, and replaces spaces with a - separator. For example, "My TEam Näme" would become my-team-name.

Reply:
Added